app/utils/whitelabel.ts

```diff
@@ -1,31 +1,89 @@
+import AskDarcelImage from '../assets/img/askdarcel.svg';
+import BackgroundImage from '../assets/img/bg.png';
+import SearchByAlgoliaImage from '../assets/img/search-by-algolia.png';
+import SFFamiliesLogo from '../assets/img/sf-families.svg';
+import SFServiceLogo from '../assets/img/sf-service.svg';
+import SFSeal from '../assets/img/sf-seal.png';
 import config from '../config';
+import icon from '../assets';
+import styles from '../components/ui/Navigation/Navigation.module.scss';
 
-/** Whether we should display the SF Service Guide branded version of the site */
-const isSFFamiliesSite = () => window.location.host.indexOf(config.SFFAMILIES_DOMAIN) > -1;
+// Include new white label here
+type WhiteLabelSiteKey = 'default' | 'SFServiceGuide' | 'SFFamilies';
 
-const isSFServiceGuideSite = () => !isSFFamiliesSite()
-  && window.location.host.indexOf(config.MOHCD_DOMAIN) > -1;
+interface WhiteLabelSite {
+  title: string;
+  siteUrl: string;
+  appImages: {
+    background: string;
+     // For appImages, logoLarge and logoSmall are usually different between each white label
+    logoLarge: string;
+    logoSmall: string;
+    algolia: string;
+    mohcdSeal: string;
+    icon: any;
+  };
+  styles?: any;
+}
 
-const getSiteTitle = () => {
-  if (isSFServiceGuideSite()) {
-    return 'SF Service Guide';
-  }
-  if (isSFFamiliesSite()) {
-    return 'SF Families';
-  }
-  return 'AskDarcel';
+// Read only to force developer to modify configurations here, disallow changes at compile time
+const configurations: Partial<Record<Readonly<WhiteLabelSiteKey>, Readonly<WhiteLabelSite>>> = {};
+
+// Include a domain in config.js
+function determineWhiteLabelSite(): WhiteLabelSiteKey {
+  if (config.SFFAMILIES_DOMAIN === window.location.host) return 'SFFamilies';
+  if (config.MOHCD_DOMAIN === window.location.host) return 'SFServiceGuide';
```

Comment:
Nit: Could you swap the order of the operands across the `===` operator, i.e. put the `window.location.host` on the left-hand side of the `===`? Even though it's logically equivalent, it's much more common in programming and in English to put the thing that is more likely to change on the left-hand side.

Reply:
ok

Reply:
ok for the WhiteLabelSiteKey
default is a reserved word so Id have to change the name

Reply:
Maybe just call it `"AskDarcel"`, since that's what it actually is?